staies/urls.py

```diff
@@ -1,6 +1,7 @@
 from django.urls  import path
-from staies.views import StayDetailView
+from staies.views import StayDetailView, StayListView
 
 urlpatterns = [
-    path('/<int:stay_id>' ,StayDetailView.as_view())
+    path('/<int:stay_id>' ,StayDetailView.as_view()),
+    path('/staylist' , StayListView.as_view())
```

Comment:
복수형태로 만들어주면 리스트라는 뜻이겠죠?
```suggestion
    path('/stays' , StayListView.as_view())
```